apps/ledger-live-desktop/src/renderer/init.jsx

```diff
@@ -60,6 +62,9 @@ async function init() {
     Transport,
   });
 
+  prepareCurrency(findCryptoCurrencyById("ethereum"));
+  preloadTokens();
```

Comment:
Agree, do specific work in the family

Reply:
by doing the prepareCurrency, it will only load dynamic token on the internal thread I guess.
I need to find a way to load tokens on the renderer thread.

Reply:
Hmmm yeah so I need to do something like : 

```
  prepareCurrency(findCryptoCurrencyById("ethereum"));
  addTokens((await hydrateCurrency(findCryptoCurrencyById("ethereum"))).erc20)
```

To have my token display in renderer, not a big fan of this. Is there any other way to load it properly on the renderer thread ?

Reply:
Nvm, I understood what was going wrong. I was doing two prepare currency ethereum, that seems to cause some issues. But I prefer the way you asked me to do now. thank you for this feedback